### DISCUSSION TITLE
Why metamask is used popular?

### DISCUSSION THREAD

**Discussion by MihalioMilosevic** [MihalioMilosevic/MetamaskFork#2] [Q&A]
Do you know why metamask is used most popular?

**Comment by CryptoSenju** [comment]
because that is easy to use.

**Reply by cryptotorpo** [reply]
Henry Scott: Hi, man, however it is better than the one!

**Comment by cryptotorpo** [comment]
However don't ask

**Reply by MihalioMilosevic** [reply]
thanks

**Comment by princekevin0x1718** [comment]
Hey, man I am right! Plz check this right now!
